tests/lib/remote.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+download_file(){
```

Comment:
Can you please make `download_file` a new executable in `tests/lib/bin`. This way it has extremely predictable behavior and does not need to be implemented in shell anymore. It also means you don't have to shell-source anything to get access to it, it is always on path.

Reply:
Also having that, we could really use python for it, dropping the need to use curl entirely.

Reply:
@zyga sure, nice idea! Thanks for reviewing!!

Reply:
Just a follow up suggestion, since we have `mountinfo-tool` and are about to get `testbed-tool`, could you please call this new one `download-tool`